bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/ConfigExtractor.scala

```diff
@@ -264,7 +266,7 @@ object ConfigExtractor {
         fieldLens <- extract[FieldLens]     (c, AoFieldLensKey)
         wfsMode   <- extract[GuideStarType] (c, AoGuideStarTypeKey)
       } yield {
-        val separation = distance(targetEnv.getBase, guideStar, when)
+        val separation = distance(targetEnv.getAsterism, guideStar.getTarget, when)
```

Comment:
Good question.  While we need the distance from the base to know if a particular guide star is viable, what matters for the calculation of the AO correction quality (Strehl) is the distance between each science target and the guide star.  The Strehl will be the highest for the target closest to the guide star and lowest for the target farthest from the guide star.